panther_analysis_tool/main.py

```diff
@@ -1048,22 +1050,39 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
     invalid_packs = validate_packs(specs)
     invalid_specs.extend(invalid_packs)
 
+    # cleanup tmp global dir
+    cleanup_global_helpers(specs[GLOBAL])
+
     print_summary(args.path, len(specs[DETECTION]), failed_tests, invalid_specs)
     return int(bool(failed_tests or invalid_specs)), invalid_specs
 
 
-def setup_global_helpers(global_analysis: List[Any]) -> List[Any]:
-    invalid_specs = []
-    for analysis_spec_filename, dir_name, analysis_spec in global_analysis:
+def setup_global_helpers(global_analysis: List[Any]) -> None:
+    # ensure the directory does not exist, else clear it
+    cleanup_global_helpers(global_analysis)
+    os.makedirs(TMP_HELPER_MODULE_LOCATION)
+    # setup temp dir for globals
+    if TMP_HELPER_MODULE_LOCATION not in sys.path:
+        sys.path.append(TMP_HELPER_MODULE_LOCATION)
+    # place globals in temp dir
+    for _, dir_name, analysis_spec in global_analysis:
         analysis_id = analysis_spec["GlobalID"]
-        module, load_err = load_module(os.path.join(dir_name, analysis_spec["Filename"]))
-        # If the module could not be loaded, continue to the next
-        if load_err:
-            invalid_specs.append((analysis_spec_filename, load_err))
-            continue
-        sys.modules[analysis_id] = module
-    return invalid_specs
-
+        source = os.path.join(dir_name, analysis_spec["Filename"])
+        destination = os.path.join(TMP_HELPER_MODULE_LOCATION, f"{analysis_id}.py")
+        shutil.copyfile(source, destination)
+        # force reload of the module as necessary
+        if analysis_id in sys.modules:
```

Comment:
I think you should be raising an exception here. Couldn't there be a conflict with a standard library module for example? For example, someone could name their helper `json` by mistake.